packages/rocketchat-ui-flextab/client/tabs/keyboardShortcuts.html

```diff
@@ -1,5 +1,5 @@
 <template name="keyboardShortcuts">
-    <div class="content">
+    <!-- <div class="content"> -->
```

Comment:
Please remove the tag instead of commenting

Reply:
Sure, will do the same.